.github/workflows/model_image_build_push.yaml

```diff
@@ -9,9 +9,15 @@ on:
 env:
   REGISTRY: quay.io/ai-lab
   MODEL_CONVERTER_IMAGE_NAME: model-converter
+  MERLINITE_MODEL_IMAGE_NAME: merlinite-7b-lab
+  MERLINITE_LABEL: Q4_K_M
+  MERLINITE_MODEL_URL: https://huggingface.co/instructlab/merlinite-7b-lab-GGUF/resolve/main/merlinite-7b-lab-Q4_K_M.gguf
   MISTRAL_MODEL_IMAGE_NAME: mistral-7b-instruct
   MISTRAL_LABEL: v0.1.Q4_K_M.gguf
   MISTRAL_MODEL_URL: https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf
+  GRANITE_MODEL_IMAGE_NAME: granite-7b-lab
+  GRANITE_LABEL: Q4_K_M
+  GRANITE_MODEL_URL: https://huggingface.co/instructlab/granite-7b-lab-GGUF/resolve/main/granite-7b-lab-Q4_K_M.gguf
```

Comment:
We should do these using our make targets for greater test coverage, see [example in ms workflows]
(https://github.com/containers/ai-lab-recipes/blob/main/.github/workflows/model_servers.yaml#L27-L46)
```suggestion
```

Reply:
matrix approach works for me! it'll be much better